audio/alaw.c

```diff
@@ -0,0 +1,126 @@
+/**
+ * @file
+ * @author [sunzhenliang](https://github.com/HiSunzhenliang)
+ * @brief A-law algorithm for encoding and decoding (16bit pcm <=> a-law).
+ * @see audio/alaw.c
```

Comment:
I think this is used when there are two or more similar files, so I don't think this is needed here. 🙂
```suggestion
```